wiki/BioSQL.md

```diff
@@ -46,38 +46,53 @@ Installing Required Software
 You will need to install some database software plus the associated
 python library so that Biopython can "talk" to the database. In this
 example we'll talk about the most common choice, MySQL. How you do this
-will also depend on your operating system, for example on a Debian or
-Ubuntu Linux machine try this:
+will also depend on your operating system.
+For example on a **Debian or Ubuntu Linux** machine try this:
 
 ``` bash
 sudo apt-get install mysql-common mysql-server python-mysqldb
 ```
-
 It will also be important to have perl (to run some of the setup
-scripts). Again, on a Debian or Ubuntu Linux machine try this:
+scripts). Again, on a **Debian or Ubuntu Linux** machine try this:
 
 ``` bash
 sudo apt-get install perl
 ```
-
 You may find perl is already installed.
 
-For Windows users, see [BioSQL on Windows](BioSQL_Windows "wikilink").
+For **Windows** users, see [BioSQL on Windows](BioSQL_Windows "wikilink").
+
+For **Cygwin** users, use [apt-cyg](https://github.com/transcode-open/apt-cyg) to install packages **mysql** and **mysql-server**,
+``` bash
+apt-cyg install mysql mysql-server
+```
+and to install the driver **mysql-connector** use [pip](https://pypi.org/project/pip/),
+
+``` bash
+pip install mysql-connector
+```
 
 Downloading the BioSQL Schema & Scripts
 ---------------------------------------
 
-Once the software is installed, your next task is to setup a database
-and import the BioSQL schema (i.e. setup the relevant tables within the
-database). See [BioSQL downloads](http://www.biosql.org/wiki/Downloads)
+Once the software is installed, your next task is to setup a database, 
+import the BioSQL schema (i.e. setup the relevant tables within the
+database) and finally populate the database.
+
+In order to do so, files from the **biosql** project need to be obtained:
+
+* Either from [BioSQL downloads](http://www.biosql.org/wiki/Downloads)
 -- you'll need to unzip the archive.
 
-Alternatively to get the very latest BioSQL, check out their git
-repository. Or, navigate to the relevant schema file for your database
-and download just that, e.g.
-[biosqldb-mysql.sql](https://raw.github.com/biosql/biosql/master/sql/biosqldb-mysql.sql)
-for MySQL. You will also want the NCBI Taxonomy loading perl script,
-[load\_ncbi\_taxonomy.pl](https://raw.github.com/biosql/biosql/master/scripts/load_ncbi_taxonomy.pl).
+* Or to get the **very latest** files, check out (or export) the relevant git
+repository at (https://github.com/biosql/biosql.git)
+
+``` bash
+svn export https://github.com/biosql/biosql.git/trunk biosql
```

Comment:
Again, avoid recommending the legacy tool ``svn`` here - we want any potential contributors to use ``git`` so:

```bash
git clone https://github.com/biosql/biosql.git
cd biosql
```

Or, for a simple snapshot:

```
wget https://github.com/biosql/biosql/archive/master.tar.gz
tar -zxvf master.tar.gz
cd biosql-master/
```